tests/integration_tests/functional/test_cmd_line_start.py

```diff
@@ -54,6 +55,12 @@ def test_config_start_with_api(test_microvm_with_ssh, vm_config_file):
     assert test_microvm.api_session.is_status_ok(response.status_code)
     assert test_microvm.state == "Running"
 
+    # Validate full vm configuration.
+    response = test_microvm.full_cfg.get()
+    assert test_microvm.api_session.is_status_ok(response.status_code)
+    with open(vm_config_file) as json_file:
+        assert response.json() == json.load(json_file)
```

Comment:
I think this should address @sandreim 's concerns about starting a VM from a full config obtained from another VM.